package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-daterange-picker",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Could we revert this line? This will allow us to control what goes into what release